apps/pie-docs/test/system/navigation.spec.js

```diff
@@ -45,3 +46,33 @@ test.describe('PIE - site nav menu - @mobile', () => {
         await expect(navMenu).not.toBeVisible();
     });
 });
+
+test.describe('PIE - site nav menu - @desktop', () => {
+    let siteNavigation;
+    let contentHeader;
+
+    test.beforeEach(async ({ page, context }) => {
+        await disableCookieBanner(page, context);
+        await page.goto('all-about-pie/what-is-pie/');
+
+        // Arrange
+        siteNavigation = await page.getByTestId('site_nav_section_1');
+        contentHeader = await page.getByTestId('content_header');
+    });
+
+    test('Should be visible at the top of the page', async () => {
+        // Assert - Navigation and top of the page are visible
+        await expect.soft(await isElementVisibleInViewport(siteNavigation)).toBeTruthy();
+        await expect.soft(await isElementVisibleInViewport(contentHeader)).toBeTruthy();
+    });
+
+    test('Should still be visible when the page has scrolled down', async ({ page }) => {
+        // Act - Scroll to the bottom of the page
+        await page.keyboard.press('End');
+        await page.waitForTimeout(1000);
+
+        // Assert - Navigation is still visible, but top of the page is not
+        await expect(await isElementVisibleInViewport(siteNavigation)).toBeTruthy();
```

Comment:
You can swap the positions of line 75 and 76 so that the siteNav expect is last and also is a hard expect. If we have a hard expect before a `.soft` expect, the test technically ends on this line without ever testing if the `contentHeader` is not visible.